pkg/startup/v4/data/master/etc/origin/node/pods/ovs.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   annotations:
     scheduler.alpha.kubernetes.io/critical-pod: ''
   labels:
-    app: ovs
+    app: ovs-static
```

Comment:
more background on this would be nice :)

Reply:
if you create a static pod with the same labels that a daemonset controller is watching, you get a loop whereby the daemonset controller keeps trying to delete the mirror pod and the kubelet keeps recreating it